elasticsearch_exporter.go

```diff
@@ -267,52 +26,52 @@ type VecInfo struct {
 
 var (
 	gaugeMetrics = map[string]string{
-		"indices_fielddata_evictions":               "Evictions from field data",
-		"indices_fielddata_memory_size_in_bytes":    "Field data cache memory usage in bytes",
-		"indices_filter_cache_evictions":            "Evictions from field data",
-		"indices_filter_cache_memory_size_in_bytes": "Field data cache memory usage in bytes",
-		"indices_docs_count":                        "Count of documents on this node",
-		"indices_docs_deleted":                      "Count of deleted documents on this node",
-		"indices_store_size_in_bytes":               "Size of stored index data in bytes",
-		"indices_segments_memory_in_bytes":          "Memory size of segments in bytes",
-		"jvm_mem_heap_committed_in_bytes":           "JVM heap memory committed",
-		"jvm_mem_heap_used_in_bytes":                "JVM heap memory used",
-		"jvm_mem_heap_max_in_bytes":                 "JVM heap memory max",
-		"jvm_mem_non_heap_committed_in_bytes":       "JVM non-heap memory committed",
-		"jvm_mem_non_heap_used_in_bytes":            "JVM non-heap memory used",
+		"indices_fielddata_evictions":            "Evictions from field data",
```

Comment:
@ewr Are you sure this is a gauge? I couldn't find any documentation about whether this is a gauge or counter. It looks like a counter to me. Same for the filter_cache_evictions below.